recipes/libalsa/all/conanfile.py

```diff
@@ -44,6 +47,7 @@ def _configure_autotools(self):
 
             args = ["--enable-static=yes", "--enable-shared=no"] \
                     if not self.options.shared else ["--enable-static=no", "--enable-shared=yes"]
+            args.append("--datarootdir=%s" % os.path.join(self.package_folder, "res", "share"))
```

Comment:
why not just res?

Reply:
in case we need to put something else in res

Reply:
I would prefer just res too. If we need to put anything else in the future (I don't think it will happen), we can update and rename to share again.

Reply:
renaming the folder would break all the consumer using this folder, but as you which